make/testing_flags.mk

```diff
@@ -61,6 +67,6 @@ backend = btcd
 endif
 
 # Construct the integration test command with the added build flags.
-ITEST_TAGS := $(DEV_TAGS) rpctest chainrpc walletrpc signrpc invoicesrpc autopilotrpc watchtowerrpc $(backend)
+ITEST_TAGS := $(DEV_TAGS) $(RPC_TAGS) rpctest $(backend)
```

Comment:
won't this result in duplicating the tags? or is the assumption that you don't need `with-rpc` for itest compilation?

Reply:
My original intention was to allow creating a dev build which includes all RPC subservers while we also keep backward compatibility with existing build targets. I'd expect `with-rpc` to be used standalone and never as part of automated builds.

Reply:
Now reading it again, it could be I'm misunderstanding your comment though :)

Reply:
nope that answers my q, agree it doesn't make sense to use `with-rpc` for itests so should be fine